Formula/pyside.rb

```diff
@@ -11,38 +11,39 @@ class Pyside < Formula
   end
 
   depends_on "cmake" => [:build, :test]
-  depends_on "llvm"
+  depends_on "llvm" => :build
   depends_on "python"
   depends_on "python@2"
   depends_on "qt"
 
   def install
+    ENV.remove "HOMEBREW_LIBRARY_PATHS", Formula["llvm"].opt_lib
```

Comment:
Why is this needed? It looks like you are trying to use `llvm` but not link against it, which is wrong. We want to avoid mismatched compiler/runtime.

Reply:
This is needed to avoid linking `pyside` to the `libc++.1.dylib` provided by `llvm`.